sites/jeromefitzgerald.com/package.json

```diff
@@ -110,7 +110,7 @@
     "eslint-config-next": "12.0.10",
     "next-compose-plugins": "2.2.1",
     "next-transpile-modules": "9.0.0",
-    "next-sitemap": "2.1.8",
+    "next-sitemap": "2.1.13",
```

Comment:
🤓  Technically this should move to `dependencies` along with the two above since it is needed for build. There is nothing currently limiting the package installs on a server environment so this is fine for now.

Reply:
Since we are here. 👍🏻